src/humanizer.ts

```diff
@@ -49,3 +49,18 @@ export const humanizeBaseMetricValue = (
       return 'Unknown';
   }
 };
+
+/**
+ * Stringify an score into a severity string ('None' | 'Low' | 'Medium' | 'High' | 'Critical')
```

Comment:
specification calls it as "qualitative severity rating"

```suggestion
 * Stringify a score into a qualitative severity rating string
```